docs/FAQ.md

```diff
@@ -107,6 +109,21 @@ behave differently:
 
 Overrides are also discussed with some visuals in [the gps docs](https://github.com/sdboyer/gps/wiki/gps-for-Implementors#overrides).
 
+## <a id="how-do-i-change-change-the-version-of-a-dependecy"></a> How do I change change the version of a dependecy
```

Comment:
Would you please remove the anchor tags? The anchor is generated from the header content and should work just fine without the anchor (like the other faq's).